src/plugins/ajs-destination/loader.ts

```diff
@@ -33,10 +33,17 @@ export async function loadIntegration(
   analyticsInstance: Analytics,
   name: string,
   version: string,
-  settings?: object
+  settings?: { [key: string]: any },
+  obfuscate?: boolean
 ): Promise<LegacyIntegration> {
   const pathName = normalizeName(name)
-  const fullPath = `${path}/integrations/${pathName}/${version}/${pathName}.dynamic.js.gz`
+  let obfuscatedPathName = ''
```

Comment:
Looks like we are repeating the same logic around `load`/`unload`. Should we consolidate in a function?

Reply:
Also nit, but might be a bit cleaner: 
```
const obfuscatedPathName = obfuscate ? btoa(pathName).replace(/=/g, '') : ''; 
```